math/wxMaxima/Portfile

```diff
@@ -1,59 +1,110 @@
 PortSystem      1.0
+
 PortGroup       active_variants 1.1
 PortGroup       wxWidgets 1.0
 
 name            wxMaxima
-version         15.04.0
-revision        0
+version             18.02.0
 maintainers     nomaintainer
-license         gpl
+license             GPL-2
+# openssl is a dependency of curl in cmake
+license_noconflict  openssl
 platforms       darwin
 categories      math aqua
-description     Graphical user interface for Maxima
-long_description \
-                wxMaxima is a cross platform GUI for the computer algebra \
-                system maxima based on wxWidgets.
```

Comment:
So, your replacement "long_description" lines remove this information. It is probably important to say it. Perhaps you should leave this in place, or add this to the long_description in each subport?